.dockerignore

```
@@ -1,2 +1,2 @@
 **/vendor
-**/.cache
+**/.cache
```

Please add new line at EOF.

```suggestion
**/.cache

```

---

[8549f25](https://github.com/acompany-develop/QuickMPC/pull/180/commits/8549f258fea615355eb12f30dd452311cb965133)

---

Thank you!